admin-bulk-user-membership/tool/src/main/resources/Messages_eu.properties

```diff
@@ -0,0 +1,47 @@
+common_title=Erabiltzaileen Kidetza-edizioa
```

Comment:
@PabloSanRoman would you mind providing translations for EU? Not in a hurry, you can take your time and I'll commit them later.